srml/consensus/src/lib.rs

```diff
@@ -216,6 +217,13 @@ decl_module! {
 			}
 		}
 
+		/// Kill some items from storage.
+		fn kill_storage(keys: Vec<Key>) {
```

Comment:
Maybe let's have `set_storage(items: Vec<(Key, Option<Value>)>)` instead?

Reply:
I like it but not sure it will work with current UI. cc @jacogr

Reply:
We have very limited vec input capabilities atm, it is not an API or metadata issue, but rather just the fact that, based on the the string type names from metadata, we have basically have "known type" custom renders for each type.

So with the proposed changed, apps won't allow input as-in just because it is limited and doesn't understand it as-is. (Can be extended specifically for this case though, so don't let current limitations stand in the way of doing the right thing, it is not an underlying limitation in the metadata)